src/core/server/http/router/validator/validator.test.ts

```diff
@@ -36,7 +36,7 @@ describe('Router validator', () => {
     expect(() => validator.getParams({})).toThrowError('[foo]: Not a string');
 
     expect(() => validator.getParams(undefined)).toThrowError(
-      "Cannot destructure property `foo` of 'undefined' or 'null'."
+      "Cannot destructure property 'foo' of 'undefined' as it is undefined."
```

Comment:
This is a change in node's error message, right?

Reply:
Correct. Technically this particular error message comes from V8 which got upgraded in Node.js 12

Reply:
https://github.com/nodejs/node/blob/6a349019da7ea79cd394101faad119db56b76bfa/deps/v8/src/common/message-template.h#L121-L122